helm/cas-postgres/Chart.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: v2
 name: cas-postgres
 description: Postgres database for the CAS team
 type: application
-version: 0.8.1
-appVersion: 0.2.0
+version: 0.8.2
+appVersion: 0.2.1
 dependencies:
   - name: patroni
-    version: "0.21.2"
+    version: "0.21.1"
```

Comment:
shouldn't this be `0.21.3` to match the changes to the other chart? or are we just reverting to the old one?